### ISSUE TITLE
If title too long text of the note in the second column overlaps (Mobile version and small screens)

### ISSUE THREAD

**Issue by ViktoriaNeva** [levelupik/levelupik#19]
Steps to reproduce:
1. Write long title, f.e.: "This is the title long title"
2. Write text more then 29 symbols
3. Lets look to the preview

Actual result:
In the small screen it will be not very pretty to look. Bottom of the text a little bit cut
![2020-05-23_11-56-21](https://user-images.githubusercontent.com/5111608/82726510-58401b80-9ced-11ea-979a-49b207d374c8.png)

But on very small screen like mobile phone it will be very cut
![380DC42F-23D7-4AE0-B0AE-49748F1996D9](https://user-images.githubusercontent.com/5111608/82726552-a1906b00-9ced-11ea-9dfb-ac66e954f65a.png)

Expected Result:
Pretty look of the text in the preview

**Comment by levelupik** [comment]
True, need to think about small screen and mobile segments, and how this preview block should look like.

Thank you,
Planned it to the future.
